include/interface.h

```diff
@@ -98,6 +98,12 @@ class Interface : public ParsedFiniteElement<dim,spacedim>
   }
 
   /**
+   * This function is used to modify triangulation using boundary_id or manifold_id.
+   * In the case a signal is required, this is the function to modify.
+   */
+  virtual void post_process_triangulation(const Triangulation<dim, spacedim> &tria) const {}
```

Comment:
`postprocess_newly_created_triangulation` ?

This name would be much more specific...